Running-Locally.md

```diff
@@ -72,17 +72,18 @@ These environment variables can be placed in the `raincatcher-demo-cloud/Gruntfi
       FH_MBAAS_PROTOCOL: "<FH_MBAAS_PROTOCOL value>",
       FH_WIDGET: "<FH_WIDGET value>",
       FH_APP_API_KEY: "<FH_APP_API_KEY value>",
+
+      // This value can be found in `raincatcher-demo-cloud/Gruntfile.js`
+      // in the FH_SERVICE_MAP environment variable
+      WFM_AUTH_GUID: "iidn3tvprs62asdebat5m3eg"
     }
   }
   ...
 }
 ```
 
+## Run The Applications
 
-### Using A Local Service To Authenticate Users
```

Comment:
@nialldonnellyfh I was just trying to simplify the docs and be clear about the configuration necessary to get everything fully running. Given that this is the "Running Locally" doc and the raincatcher CLI automatically starts up the auth service, I find it unnecessary to say "The *raincatcher-demo-auth* service can be run locally for ease of editing".

I also had to read this section several times and dig through gruntfile.js to find the GUID this section was talking about. I thought that putting the environment variable directly in the README (see https://github.com/feedhenry-raincatcher/raincatcher-documentation/pull/5/files#diff-a6455134ad1554da1161df5b89e71eacR76) would help eliminate some of that extra mental work for users getting started.

Does that make sense? I can roll back the changes if you strongly disagree.